recipes/cyrsoxs/build.sh

```diff
@@ -0,0 +1,12 @@
+
+mkdir build
+cd build
+cmake ${SRC_DIR} ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release 
+make install
+cd ../
+
+mkdir build-pybind
+cd build-pybind
+cmake ${SRC_DIR} ${CMAKE_ARGS} -DCMAKE_BUILD_TYPE=Release -DPYBIND=Yes -DUSE_SUBMODULE_PYBIND=No -DPython_EXECUTABLE="$PYTHON"  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
```

Comment:
its often really useful to just have one option per line once they get this big. this doesn't even fit on my screen.

Reply:
Done, thanks.